package.json

```diff
@@ -29,7 +29,7 @@
     "refapp3Settings": "cypress run --spec src/test/resources/features/refapp-3.x/03-settings/settings.feature",
     "refapp3ClinicalVisit": "cypress run --spec src/test/resources/features/refapp-3.x/04-clinical-visit/clinical-visit.feature",
     "refapp3VitalsAndTriage": "cypress run --spec src/test/resources/features/refapp-3.x/05-vitals-and-triage/vitals-and-triage.feature",
-    "refappOclSubscriptionModule": "cypress run --spec src/test/resources/features/oclsubscriptionmodule/index.feature"
+    "refappOclSubscriptionModule": "cypress run --spec src/test/resources/features/oclsubscriptionmodule/oclsubscriptionmodule.feature"
```

Comment:
It would be great if we have consistency in cypress naming of folders, js file and feature files. I would rename the package to `ocl-subscription-module`, js file to `ocl-subscription-module.js` and the feature file to `ocl-subscription-module.feature`. 

@hadijahkyampeire what is your thought?

Reply:
Sure, this sounds great.